src/hotspot/cpu/aarch64/matcher_aarch64.hpp

```diff
@@ -198,6 +198,10 @@
     }
   }
 
+  static bool is_vector_cast_supported(BasicType from_bt, BasicType to_bt) {
+    return false;
+  }
```

Comment:
Do these simply not exist, or do you just want to leave that to a future RFE so someone else can take care of this?